docs/instrumentation-list.yaml

```diff
@@ -5300,8 +5300,12 @@ libraries:
   - name: jaxrs-1.0
     display_name: JAX-RS 1.x
     description: |
-      This instrumentation enables spans for JAX-RS methods, and enriches existing HTTP server spans with route information.
+      This instrumentation enriches HTTP server spans with route information and enables controller spans for JAX-RS annotated methods (this instrumentation is disabled by default).
```

Comment:
JAX-RS 2.x uses a slightly different wording `controller spans are disabled by default` which I think is better